scripts/third_party_size_check.py

```diff
@@ -33,7 +33,7 @@
 import yaml  # isort:skip  #pylint: disable=wrong-import-position
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
-THIRD_PARTY_SIZE_LIMIT = 8500
+THIRD_PARTY_SIZE_LIMIT = 12000
```

Comment:
Why is this increased? This can cause problems when deploying on the production.

Reply:
This was increased because adding the apache beam dependency was causing the third party size check to fail. we might be able to decrease it again when we remove the MapReduce dependency

Reply:
Just noting that this should now be OK from a release standpoint (and if not we'll find out during testing!).

Reply:
IIRC, the limit on GAE upload was around 10,000 files. [not sure!]

@shubha-rajan, do we need the limit to be 1200 or it can be reduced further?

Reply:
That's correct, but I looked into increasing this, so we should be fine, as long as it doesn't go much above 12,000.

Reply:
For now, I think 12,000 is as low as we can go:
```
Running third-party size check

------------------------------------------------------
    Number of files in third-party folder: 11922

    ERROR: The third-party folder size exceeded the 8500 files limit.
------------------------------------------------------
```

Reply:
Okay, if Sean confirms it is fine I have no problem.